packages/tabs/docs/Tabs.stories.tsx

```diff
@@ -145,7 +139,7 @@ export const UsageInCard = () => (
       // eslint-disable-next-line no-console
       onChange={index => console.log("Tab changed to ", index)}
     >
-      <Box pt={0.25} pl={1} pr={1}>
+      <Box pt={0.25}>
```

Comment:
Question is, should this padding be built in as well 🤔

Reply:
It would probably be easier to reason about if all padding was built in, especially as the prop is named "unpadded"